### PR TITLE
more explanation of github administration

### PR DESCRIPTION
**Pull request by ultrasaurus**
addresses issue #543

### CODE REVIEW

File: governance/github.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Github access permissions and administration
+
+Facilitation roles are identified in [github settings](/.github/settings.yml) 
+which we use for Github admin permissions and managing issues.  Write 
+permissions are enabled by the [CODEOWNERS](/CODEOWNERS) file.
+
+There is typically more process for review and collaboration than is controlled
+by access permissions. We expect members to review [governance](governance)
```
Comment (ashutosh-narkar) [inline governance/github.md:8]: The `governance` link is broken.

File: governance/github.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+and ask questions by filing a Github issue and/or submit suggested changes via
+Pull Request if anything is not clear.
+
+Chairs have admin privileges and have access to change settings in the Github
+UI.  Except where noted below, changes should be made in the repo files to
+control access privileges, not in the Github UI (so they are visible to
+everyone.)  
+
+Note: Members of the CNCF TOC and some CNCF staff also have admin access;
+however,
```
Comment (ashutosh-narkar) [inline governance/github.md:18]: Nit: Were you adding more to this statement ?

**Review by ashutosh-narkar — APPROVED** [review]
Couple of nits but LGTM.

File: governance/github.md (hunk 1 above)
Comment (ultrasaurus) [reply governance/github.md]: fixed

File: governance/github.md (hunk 2 above)
Comment (ultrasaurus) [reply governance/github.md]: oops -- thanks! I finished the sentence :)

**Pull request merged**